.github/templates/linux_ci_workflow.yml.in

```diff
@@ -0,0 +1,140 @@
+# WARNING THIS IS AUTOGENERATED DO NOT UPDATE MANUALLY PLS
```

Comment:
Just use the `@generated` convention. Have a variable that expands to this, and then spell it in Python like `"@" + "generated"` so the codegen file doesn't treat it as generated.